main.py

```
@@ -162,7 +162,7 @@ def create_output_folder() -> Path:
     return output_path
 
 
-def setup_logging(output_path: Path, debug_level=False):
+def setup_logging(output_path: Path, debug_level: bool = False) -> None:
```

Ruff (0.14.0)

Boolean-typed positional argument in function definition

(FBT001)